tidy3d/plugins/near2far/near2far.py

```diff
@@ -278,7 +545,7 @@ def _sph_2_car(r, theta, phi):
         theta : float
             polar angle (rad) downward from x=y=0 line
         phi : float
-            azimuthal (rad) angle from x=z=0 line
+            azimuthal (rad) angle from y=z=0 line
```

Comment:
This still true?  And is it rather y=z=self.origin[0]?  Just confirm whether these docstrings need to be modified.

Reply:
Yeah good point, I will need to modify these